ReactSkia/components/RSkComponentImage.h

```diff
@@ -8,7 +14,10 @@
 #include "ReactSkia/views/common/RSkImageCacheManager.h"
 
 #define DEFAULT_IMAGE_FILTER_QUALITY kLow_SkFilterQuality /*Skia's Defualt is kNone_SkFilterQuality*/
-
+#define DEFAULT_MAX_CACHE_EXPIRY_TIME 1800000 // 30mins in seconds 1800000
+#define NO_CACHE "no-cache"
+#define NO_STORE "no-store"
+#define MAX_AGE "max-age=0"
```

Comment:
this macro name should be MAX_AGE_0  ?
also need to add macro for "max-age" string